pycode/memilio-epidata/memilio/epidata_test/test_epidata_getCommuterMobility.py

```diff
@@ -337,94 +110,70 @@ def test_assign_geographical_entities(self, mock_print):
         Errorcall = ('Error. Number of government regions wrong.')
         mock_print.assert_called_with(Errorcall)
 
-    @patch('builtins.print')
```

Comment:
why delete this test?

Reply:
Because it doesn't test the getCommuterMobility.py